api/src/main/java/javax/enterprise/inject/spi/builder/Builders.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2016, Red Hat, Inc., and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package javax.enterprise.inject.spi.builder;
+
+import java.util.ServiceLoader;
+
+/**
+ * Provide static access to various CDI builders.
+ * Will load {@link BuildersProvider} implementation with {@link ServiceLoader}.
+ *
+ *
+ * @see BuildersProvider
+ * @author Antoine Sabot-Durand
+ * @since 2.0
+ */
+public final class Builders {
+
+    /**
+     * {@link BuildersProvider} instance loader from {@link ServiceLoader}
+     */
+    protected static volatile BuildersProvider buildersProvider = null;
```

Comment:
To ease tests.
It's like the fields in CDI class...
What visibility did you expect?